diff --git a/pex/pex.py b/pex/pex.py
index 138984f..1618968 100644
--- a/pex/pex.py
+++ b/pex/pex.py
@@ -429,7 +429,7 @@ class PEX(object):  # noqa: T000
   @staticmethod
   def execute_module(module_name):
     import runpy
-    runpy.run_module(module_name, run_name='__main__')
+    runpy.run_module(module_name, run_name='__main__', alter_sys=True)
 
   @staticmethod
   def execute_pkg_resources(spec):
